src/tpm20_compat.h

```diff
@@ -21,14 +21,25 @@
 
 #include <stdio.h>
 
-#include <sapi/tpm20.h>
-
-#ifdef TCTI_DEVICE_ENABLED
-#include <tcti/tcti_device.h>
-#endif // TCTI_DEVICE_ENABLED
-#ifdef TCTI_TABRMD_ENABLED
-#include <tcti/tcti-tabrmd.h>
-#endif // TCTI_TABRMD_ENABLED
+#ifndef TSS_COMPAT
+#  include <tss2/tss2_sys.h>
+#  include <tss2/tss2_tcti.h>
+#  include <tss2/tss2_tcti_device.h>
+#  include <tss2/tss2_tcti_mssim.h>
+#  ifdef TCTI_TABRMD_ENABLED
+#    include <tcti/tss2-tcti-tabrmd.h>
```

Comment:
<tcti/tss2-tcti-tabrmd.h> may change in the future

I have posted an issue https://github.com/tpm2-software/tpm2-abrmd/issues/395 to the upstream develpoers.